bin/BuildPackages.sh

```diff
@@ -86,6 +105,28 @@ then
   GAPDIR="$(pwd)"
   popd
 fi
+
+# check if coloring should be turned off
+if [[ "$#" -ge 1 ]]
+then
+  case "$1" in
+    --no-color|--no-colors|--no-colour|--no-colours|\
+    --nocolor|--nocolors|--nocolour|--nocolours)
+      COLOR=0
+      shift
+    ;;
+  esac
+fi
```

Comment:
I thought about that, the only (minor) problem is then you have to worry about making one script find the other, which is a pain cross-platform. This single script could still have cleaner separation, with different functions.